StudentAPI/Services/StudentService.cs

```diff
@@ -14,14 +14,20 @@ public interface IStudentService
     }
     public class StudentService : IStudentService
     {
+        private readonly DataContext _context;
+
+        public StudentService()
+        {
+            _context = new DataContext();
```

Comment:
Use DI container to inject services. 
```suggestion
        public StudentService(DataContext context)
        {
            _context = context;
```